packages/mongodb-runner/src/mongocluster.spec.ts

```diff
@@ -343,4 +345,178 @@ describe('MongoCluster', function () {
       baddoc: 1,
     });
   });
+
+  it('can pass custom arguments for replica set members', async function () {
+    cluster = await MongoCluster.start({
+      version: '6.x',
```

Comment:
The other tests are also using it, a bit of an unrelated question, should we be using an [EOL server](https://www.mongodb.com/legal/support-policy/lifecycles) here? Should we have these new tests use a newer version like 8.x?

Reply:
Yeah, I think we can bump it, the main reason I copied 6.x is that adding new versions in additions to the old ones would lead to more time spent downloading